Index: ButentWS.java
===================================================================
--- ButentWS.java	(revision 1551)
+++ ButentWS.java	(working copy)
@@ -27,9 +27,9 @@
 @WebServiceClient(name = ButentWS.NAME, targetNamespace = ButentWS.NAMESPACE)
 public class ButentWS extends Service {
 
-  static final String NAMESPACE = "http://localhost/ButentWS/wsdl/";
+  static final String NAMESPACE = "http://localhost/Butent1WS/wsdl/";
   static final String PORT = "ButentWebServiceSoapPort";
-  static final String NAME = "ButentWS";
+  static final String NAME = "Butent1WS";
   static final String ACTION = "http://localhost/ButentWS/action/ButentWebService";
 
   private static BeeLogger logger = LogUtils.getLogger(ButentWS.class);
